storage_service/static/js/project.js

```diff
@@ -27,4 +27,34 @@ $(document).ready(function() {
           },
       }
     });
-} );
+
+    $("a.request-delete").click(function() {
+        var self = $(this);
+        var uuid = self.data('package-uuid');
+        var pipeline = self.data('package-pipeline');
+        var $userDataEl = $("#user-data-packages");
+        var userId = $userDataEl.data('user-id');
+        var userEmail = $userDataEl.data('user-email');
+        var userUsername = $userDataEl.data('user-username');
+        var userAPIKey = $userDataEl.data('user-api-key');
+        var formData = {
+            "event_reason": "Storage Service user wants to delete AIP " + uuid + ".",
+            "pipeline": pipeline,
+            "user_id": userId,
+            "user_email": userEmail};
+        $.ajax({
+            type: "POST",
+            url: "/api/v2/file/" + uuid + '/delete_aip/',
+            data: JSON.stringify(formData),
+            dataType: "json",
+            contentType: "application/json; charset=utf-8",
+            headers: {"Authorization": "ApiKey " + userUsername + ":" + userAPIKey},
+            success: function(data) {
+                alert(data.message);
```

Comment:
Alerts are considered harmful, should we avoid them?
Maybe we can file a bug to remember?

Reply:
Or we could have the JS create an alert that looks like the Django one we use elsewhere...

Reply:
Makes sense. There is already a message framework in Django to do this but I guess you can't use it because this is now happening on the client side.